pt/deploy/README.md

```diff
@@ -188,8 +188,8 @@ Assim como fez em seu próprio computador, você pode criar um virtualenv na Pyt
     {% filename %}PythonAnywhere command-line{% endfilename %}
     20:20 ~ $ cd my-first-blog
 
-    20:20 ~ $ virtualenv --python=python3.6 myvenv
-    Running virtualenv with interpreter /usr/bin/python3.6
+    20:20 ~ $ virtualenv --python=python3.5 myvenv
+    Running virtualenv with interpreter /usr/bin/python3.5
```

Comment:
We probably don't want this change

Reply:
@priromagnoli Indeed, please change it back.

Reply:
There is another PR that corrects this, so I'll merge this one and the other afterwards so it will be fixed.

Reply:
Actually, @aniav, now I get it.

The install instructions in portuguese are in Python 3.5, so this makes sense (unless we change that instead).

Reply:
@rsip22 Yes... This was the reason why I've changed it, Python version is defined as 3.5 in PythonAnywhere. I'm sorry for not explaining this before!